src/pages/iou/request/step/IOURequestStepTaxAmountPage.tsx

```diff
@@ -151,7 +151,7 @@ function IOURequestStepTaxAmountPage({
                 isEditing={Boolean(backTo || isEditing)}
                 currency={currency}
                 amount={transactionDetails?.taxAmount}
-                taxAmount={getTaxAmount(transaction, policy, Boolean(backTo || isEditing))}
+                taxAmount={getTaxAmount(transaction, policy, currency, Boolean(backTo || isEditing))}
```

Comment:
I believe we can also leave this? As here we're displaying the taxAmount associated with transaction currency.

Reply:
Ah okay 👍